### DISCUSSION TITLE
Fantastic video annotation example but cannot find it anymore

### DISCUSSION THREAD

**Discussion by 99bits** [katspaugh/wavesurfer.js#3365] [Q&A]
Thank you for this amazing project.  

I had seen this example before and it was amazing.  I tried to find it but somehow it's taken down.
Would it be possible to reinstate this example again please.  

<img width="859" alt="2023-11-29_22-22-41" src="https://github.com/katspaugh/wavesurfer.js/assets/22409112/0a5ef8ac-c572-4ed1-9907-c900f8feffee">

**Comment by katspaugh** [comment]
Looks like it has to be updated for v7:
https://wavesurfer.xyz/example/video-annotation/

**Reply by 99bits** [reply]
Awesome, that's exactly the example.  Happy to see it again because it's a very cool example that demonstrates how wavesurfer can be used in more complex real world use cases with videos.   Would it take long for this example to be working again on version 7?

**Reply by katspaugh** [reply]
You can go ahead and update it if you want it to happen faster.

**Reply by 99bits** [reply]
Would love to but I'm quite a newbie and it would take me a long time to learn both JS and wavesurfer.

**Reply by 99bits** [reply]
Is there a link to the github code for this example?

**Reply by katspaugh** [reply]
Yeah, it's on the v6 branch: https://github.com/katspaugh/wavesurfer.js/tree/v6/example/video-annotation-contentEditable-regions
